prow/plugins/trigger/ic.go

```diff
@@ -100,25 +100,25 @@ func handleIC(c client, trustedOrg string, ic github.IssueCommentEvent) error {
 
 	var comments []github.IssueComment
 	// Skip untrusted users.
-	orgMember, err := isUserTrusted(c.GitHubClient, commentAuthor, trustedOrg, org)
+	trusted, err := trustedUser(c.GitHubClient, trigger, commentAuthor, org, repo)
 	if err != nil {
-		return err
+		return fmt.Errorf("error checking trust of %s: %v", commentAuthor, err)
 	}
-	if !orgMember {
+	if !trusted {
 		comments, err = c.GitHubClient.ListIssueComments(org, repo, number)
 		if err != nil {
-			return err
+			return fmt.Errorf("error listing issue comments: %v", err)
 		}
-		trusted, err := trustedPullRequest(c.GitHubClient, *pr, trustedOrg, comments)
+		trusted, err := trustedPullRequest(c.GitHubClient, trigger, commentAuthor, org, repo, comments)
 		if err != nil {
 			return err
 		}
 		if !trusted {
 			var more string
-			if org != trustedOrg {
+			if trigger != nil && trigger.TrustedOrg != org {
 				more = fmt.Sprintf("or [%s](https://github.com/orgs/%s/people) ", org, org)
```

Comment:
If `more` is specified we will link to `org` twice instead of mentioning both `org` and `trustedOrg` so `trustedOrg` should be used here instead of `org`.

Reply:
Done